testing/asserts.ts

```diff
@@ -714,13 +707,6 @@ export function assertRejects<E extends Error = Error>(
   msgIncludes?: string,
   msg?: string,
 ): Promise<E>;
-/** @deprecated (will be removed after 0.157.0) Use assertRejects(fn, msg) instead, which now returns rejected value
- * and you can assert on it. */
-export function assertRejects(
-  fn: () => PromiseLike<unknown>,
-  errorCallback: (e: Error) => unknown,
-  msg?: string,
-): Promise<Error>;
 export async function assertRejects<E extends Error = Error>(
   fn: () => PromiseLike<unknown>,
   errorClassOrCallbackOrMsg?:
```

Comment:
This variable now can be much simpler like `errorClassOrMsg: (new (...args: any[]) => E) | string`.

Also `let errorCallback` declaration below (and other related stuff) can be removed.